### PR TITLE
change project name; bibtex-actions -> citar

### PR DESCRIPTION
**Pull request by bdarcus**
### todo

- [ ] update screenshots  
- [ ] update version and changelog 
- [ ] publicize
- [x] submit new recipe to melpa
- [x] change all file and symbol names
- [x] update readme

### CODE REVIEW

**Comment by bdarcus** [comment]
@minad - here's the commit that removes the embark require line, etc. Is this right?

https://github.com/bdarcus/bibtex-actions/pull/349/commits/5acfc9fd14fbf12637ebb6944425ea29f744275a

**Comment by minad** [comment]
Seems about right. There is another reference to embark-act though. I am not yet sure what to make about that. I'd like to give your package a run soon, but I am still waiting for the rename and #307 :)

**Comment by bdarcus** [comment]
> Seems about right. There is another reference to embark-act though. I am not yet sure what to make about that. 

Oh right; in the keymap?

Hmm. I guess that was kind of why I was wanting to require embark, so that a user could right-click on an org citation and get the menu of commands without having to configure it themselves.

> I'd like to give your package a run soon, but I am still waiting for the rename and #307 :)

I just submitted the melpa PR.

https://github.com/melpa/melpa/pull/7781

File: citar-org.el [hunk 1]
```diff
@@ -27,103 +27,102 @@
...
-  :group 'oc-bibtex-actions
+  :group 'citar-org
   :type '(restricted-sexp :match-alternatives (keymapp)))
 
-(defcustom oc-bibtex-actions-citation-map
+(defcustom citar-org-citation-map
   (let ((map (make-sparse-keymap)))
-    (define-key map (kbd "<mouse-1>") '("default action" . bibtex-actions-dwim))
+    (define-key map (kbd "<mouse-1>") '("default action" . citar-dwim))
     (define-key map (kbd "<mouse-3>") '("embark act" . embark-act))
```
Comment (bdarcus) [inline citar-org.el:109]:
```suggestion
    (with-eval-after-load 'embark
      (define-key map (kbd "<mouse-3>") '("embark act" . embark-act)))
```

Can I do this @minad?

And is it a problem that I get these warning using `with-eval-after-load`?

```elisp
Compiling file /home/bruce/Code/bibtex-actions/citar-org.el at Sun Oct 31 13:38:47 2021
citar-org.el:334:17: Warning: reference to free variable
    ‘embark-target-finders’
citar-org.el:334:17: Warning: assignment to free variable
    ‘embark-target-finders’
citar-org.el:335:17: Warning: reference to free variable ‘embark-keymap-alist’
citar-org.el:336:17: Warning: assignment to free variable
    ‘embark-keymap-alist’
citar-org.el:337:17: Warning: reference to free variable
    ‘embark-pre-action-hooks’
citar-org.el:337:17: Warning: assignment to free variable
    ‘embark-pre-action-hooks’
```

Comment (minad) [reply citar-org.el:109]: You can declare this variables. Then then warnings will go away.

~~~ elisp
(defvar embark-pre-action-hooks)
...
~~~

**Pull request merged**